chrome/extension/ergo-connector/connect.js

```diff
@@ -45,7 +45,8 @@ chrome.runtime.sendMessage({ type: 'connect_retrieve_data' }, async response =>
     connect.onclick = () => {
       const accountsSelected: HTMLCollection<HTMLInputElement> = (document.getElementsByName('account'): any);
       for (let i = 0; i < accountsSelected.length; i += 1) {
-        if (accountsSelected[i].checked === true) {
```

Comment:
Is this file even still being used at all with the new UI?

Reply:
cc @paulclindo

Reply:
@rooooooooob Good catch! I've just removed the HTML and JS files since we don't need them in the new UI